gratipay/utils/team_history.py

```diff
@@ -0,0 +1,190 @@
+from datetime import datetime
+from decimal import Decimal as D
+
+from aspen import Response
+
+
+def get_end_of_year_totals(db, team, year):
+
+    received = db.one("""
```

Comment:
This came up at https://github.com/gratipay/gratipay.com/pull/4190#discussion_r88921463 as well.